cmd/infra/powervs/create.go

```diff
@@ -375,37 +373,48 @@ func (infra *Infra) setupBaseDomain(options *CreateInfraOptions) (err error) {
 
 			if zoneList != nil {
 				for _, zone := range zoneList.Result {
-					if *zone.Name == options.BaseDomain {
-						infra.CisCrn = *resource.CRN
-						infra.CisDomainID = *zone.ID
-						isDone = true
-						return
+					if *zone.Name == baseDomain {
```

Comment:
is the name a universal id or there might be more than one with same name?

Reply:
There is no possibility for dup name to exist. Name will be unique under a CIS service ID.